x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/aggregate/Avg.java

```diff
@@ -55,11 +58,16 @@ public Avg(
             description = "Expression that outputs values to average."
         ) Expression field
     ) {
-        this(source, field, Literal.TRUE);
+        this(source, field, Literal.TRUE, SummationMode.COMPENSATED_LITERAL);
```

Comment:
Should we consider also using `LOSSY_LITERAL` for `avg` the function is used in the context of `TS` source command? If so, then maybe we can do that in a followup?